scss/components/side-nav.scss

```diff
@@ -25,7 +25,7 @@ $block: #{$fd-namespace}-side-nav;
     $fd-side-nav-hover-background-color: fd-color("neutral", 1);
     $fd-side-nav-link-color: fd-color("text", 2);
     $fd-side-nav-max-width: 249px;
-    $fd-side-nav-link-padding: fd-space(2.5) fd-space(5);
+    $fd-side-nav-link-padding: fd-space(2.5) fd-space(7.5) fd-space(2.5) fd-space(5);
```

Comment:
Not sure the right padding is large enough yet.  See screenshot:
<img width="1602" alt="Screen Shot 2019-06-24 at 12 20 31 PM" src="https://user-images.githubusercontent.com/11407353/60038745-bc400700-967a-11e9-8a2a-209f39f3b796.png">